gradle/root/coverage.gradle

```diff
@@ -1,3 +1,11 @@
+if (!project.hasProperty("jacoco")) {
+    //JaCoCo plugin prevents using Build Cache. We will enable jacoco configuration only when specific property is supplied
+    logger.info "JaCoCo code coverage will not be configured because 'jacoco' project property is not present."
+    return //don't evaluate this Gradle file any further
+} else {
+    logger.lifecycle "Configuring JaCoCo code coverage because 'jacoco' project property is present."
+}
```

Comment:
I'm ok with this change. 
The other option would be to set `tasks.test.jacoco.enabled = false`.
In this case we don't have to add another parameter to the command line.

Reply:
Great! The workaround in this PR is cheap so I'll merge it. Happy to remove it when we migrate to Gradle 5.0. I'll add a note.